gengo/languages.yaml

```diff
@@ -219,6 +219,25 @@ Erlang:
     extensions:
       - erl
       - hrl
+"FORTRAN Legacy":
```

Comment:
Any reason why one is UPPERCASE and the other is Titlecase?

Reply:
Modern fortran is mostly stylised as "Fortran" (as a name) in books/promotional images and whatnot, whereas legacy fortran was almost always stylised as "FORTRAN" (as an abbreviation for "FORmula TRANslation"), so I was following that convention.

Reply:
Got it, thanks!